static/app/utils/replays/hooks/useReplayData.tsx

```diff
@@ -128,58 +150,48 @@ function useReplayData({
       perPage: segmentsPerPage,
     });
 
-  const getErrorsQueryKey = useCallback(
-    ({cursor, per_page}): ApiQueryKey => {
-      // Clone the `finished_at` time and bump it up one second because finishedAt
-      // has the `ms` portion truncated, while replays-events-meta operates on
-      // timestamps with `ms` attached. So finishedAt could be at time `12:00:00.000Z`
-      // while the event is saved with `12:00:00.450Z`.
-      const finishedAtClone = new Date(replayRecord?.finished_at ?? '');
-      finishedAtClone.setSeconds(finishedAtClone.getSeconds() + 1);
+  const startedAt = useMemo(() => {
+    return dateOffset(new Date(replayRecord?.started_at ?? ''), START_TIME_FUZZ_MS);
```

Comment:
is `new Date('')` the same as `new Date()`?

Reply:
Not the same, the values are different:

![SCR-20240430-nhvu](https://github.com/getsentry/sentry/assets/187460/e8e54ccd-71c1-4f58-9508-c5db7239d290)


But! they are both `instanceof Date`

Reply:
is there a chance that `replayRecord.started_at` or `replayRecord.finished_at`? could be undefined, leading to an error when we try to render the dates?

Reply:
yup, that's why the ` ?? ''` is in there

We're taking `replayRecord?.started_at` which is typed as `Date | undefined` and cloning it by passing it back into `new Date()`... so it'll be a `Date` in the end for sure. It'll either be a reasonable value like `2024-04-30`, or it'll be the `Invalid Date` date (like how NaN is a valid number)
It's important to clone here because we don't want to change the replayRecord, just use different times for the queries below.

Reply:
gotcha, makes sense